### ISSUE TITLE
Abilities to reset Mixer Panel Settings to Default / Revert Mixer Panel changes to "new-score" layout. (i.e. All Volumes reset to 0 etc...)

### ISSUE THREAD

**Issue by UncleRed0899** [musescore/MuseScore#24685]
### Your idea

It would be helpful if there were a way to revert all changes in the Mixer Workspace Panel back to "New-Score" defaults **_(All 0 volume, 0 L/R Panning, 0 Reverb, and Empty Aux Channels (aside from the Musesound Default MuseReverb Aux Channel 1, as would be default for a new score))_**

### Problem to be solved

This would resolve the issue of a User having to reset all sounds, panning, and sound FX settings to 0, and/or remove all Aux Channel settings, manually, by systematically going through each and every instrument sound, 1 by 1, to complete this task. **_(Especially on Music Scores that have 10+ individual parts)_**

### Additional context

Ran search queries for this issue/suggestion, and was unable to locate any. If I skimmed it over, please disregard this issue. I am a _**NEW**_ New user to Github, and apologize for any mistakes made during this issue / feature request drafting.

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by bkunda** [comment]
Agree, but we should probably ensure we implement undo actions for the mixer beforehand (so the user can if necessary return to the previous customised state after resetting everything).

**Comment by wjana298** [comment]
Hi! I implemented the button (upper right corner) and it looks like this. I also added a pop-up window in order to replace the undo option for now. What do you think?

<img width="1919" height="513" alt="Image" src="https://github.com/user-attachments/assets/e54c218f-ab1b-4ff3-863a-61794860837f" />

<img width="1919" height="380" alt="Image" src="https://github.com/user-attachments/assets/228d728b-45b7-4c7c-a12b-066a931fb669" />
